### ISSUE TITLE
Several missing features

### ISSUE THREAD

**Issue by LozoAntoha** [knightss27/grafana-network-weathermap#57]
Hello. Kudos to this plugin. I would be extremely grateful if the link data was displayed in the pop-up window. With a large number of hosts, this is necessary =)
Also, there is not enough scalability so that you can zoom in and out a certain section of the map in the widget.

**Comment by knightss27** [comment]
Could you be more specific? I'm not sure what you mean about the scalability.

**Comment by LozoAntoha** [comment]
It would be convenient in the dashboard to zoom in on a certain zone of the network map and move it away

**Comment by knightss27** [comment]
Not sure if this is what you mean, but you can zoom on the map (outside of edit mode) already? Holding `Shift` + scrolling or dragging will let you zoom and move around (even as a read-only user).

**Comment by knightss27** [comment]
Or do you mean to be able to select an entire portion of the map to move at once? That would make more sense.

**Comment by LozoAntoha** [comment]
> Not sure if this is what you mean, but you _can_ zoom on the map (outside of edit mode) already? Holding `Shift` + scrolling or dragging will let you zoom and move around (even as a read-only user).


great, didn't know

**Comment by LozoAntoha** [comment]
And do not tell me what about the pop-up window?
And I'm also interested - in one of the updates, it was indicated that you can bend lines between network nodes, I can't find it. Or is it planned?

**Comment by knightss27** [comment]
>  And I'm also interested - in one of the updates, it was indicated that you can bend lines between network nodes, I can't find it. Or is it planned?

Yes, please see the wiki: https://grafana-weathermap.seth.cx/customization/#advanced

**Comment by LozoAntoha** [comment]
Hello.
Thanks, got it.

Can I change adding one more metric? Description of the port.
It would be convenient to see this data in a pop-up window.
